packages/client/src/schema/pagination.ts

```diff
@@ -127,6 +130,14 @@ export class RecordArray<Result extends XataRecord> extends Array<Result> {
     return new Array(...args);
   }
 
+  toArray(): Result[] {
+    return new Array(...this);
+  }
+
+  map<U>(callbackfn: (value: Result, index: number, array: Result[]) => U, thisArg?: any): U[] {
```

Comment:
Any other method that we should consider overwriting? I don't think so, right?

Reply:
`forEach` doesn't return anything and `reduce` has the return type in the method signature so doesn't need us to change it. I can't think of others to be honest, let's see what users complain in the future 💥